Examples/Io/Root/src/RootTrackSummaryWriter.cpp

```diff
@@ -44,9 +45,10 @@ using Acts::VectorHelpers::perp;
 using Acts::VectorHelpers::phi;
 using Acts::VectorHelpers::theta;
 
-ActsExamples::RootTrackSummaryWriter::RootTrackSummaryWriter(
-    const ActsExamples::RootTrackSummaryWriter::Config& config,
-    Acts::Logging::Level level)
+namespace ActsExamples {
```

Comment:
should we maybe also change the namespace-usage in the source-files of the other readers/writers?

Reply:
I think all our source files should look like this. For free functions it makes sense to bind them explicitly to the namespace IMO.

I am adopting this in the source files I touch lately but didn't plan on a large scale refactor so far